ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -834,6 +834,10 @@ router bgp {{ router_bgp.as }}
 {%                 set redistribute_route_cli = "redistribute " ~ redistribute_route.source_protocol %}
 {%                 if redistribute_route.route_map is arista.avd.defined %}
 {%                     set redistribute_route_cli = redistribute_route_cli ~ " route-map " ~ redistribute_route.route_map %}
+{%                 elif redistribute_route.source_protocol == "isis" %}
```

Comment:
For isis redistribute_route default it takes level-2 however we have the option to set on different level. Can you check in your EOS version that it has level visibility or not
```
router bgp 65000
   address-family ipv4 multicast
      redistribute isis level-2 route-map ROUTMAP
```

Reply:
Please don't bloat this PR. It is about RCF, so just implement what we do elsewhere for redistribution.

Reply:
Won't be addressing in this PR